src/common/types.ts

```diff
@@ -14,8 +14,11 @@ interface ColumnValue<TRow, TField extends keyof TRow = keyof TRow> {
   width?: number | string;
   cellClass?: string | ((row: TRow) => string);
   headerCellClass?: string;
+  summaryCellClass?: string | ((row: any) => string);
```

Comment:
We are adding similar props for each section (header, filter, body, and footer)
- height, headerRowHeight, headerFiltersHeight
- cellClass, headerCellClass, summaryCellClass

I wonder if there is a better way to organize props so we do have to repeat them 🤔